packages/styles/scss/patterns/blocks/logo-grid/_logo-grid.scss

```diff
@@ -27,49 +25,51 @@
         margin: $layout-05 0;
       }
     }
-    &__container {
-      border-bottom: 1px $ui-04 solid;
-    }
-
-    &__no-border {
-      .#{$prefix}--logo-grid__container {
-        border-bottom: none;
-      }
-    }
 
     .#{$prefix}--card__wrapper {
       width: 100%;
     }
 
     .#{$prefix}--content-block__cta {
       margin-top: $layout-01;
-      margin-bottom: $layout-07;
-      @include carbon--make-col-ready;
-      @include carbon--make-col(1, 1);
-
       padding-left: $spacing-05;
 
-      @include carbon--breakpoint('md') {
-        @include carbon--make-col(1, 3);
+      &.#{$prefix}--content-block__cta-col {
+        @include carbon--breakpoint('md') {
+          padding-left: 0;
+          padding-right: 0;
+        }
       }
 
       .#{$prefix}--card__heading {
         @include carbon--type-style('productive-heading-01');
       }
     }
 
-    a.#{$prefix}--card__CTA {
-      max-width: 100%;
-      margin-left: 0;
-    }
-
     &__link {
```

Comment:
Starting here started outside of your component class, so you may want to start looking here.